scripts/nb_filter_smudge.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+export PYTHONPATH="${PWD}/code"
```

Comment:
If the host system has some `PYTHONPATH` already set, this will erase it, in particular if someone is using virtual environments. The correct way is to append to the variable:
```bash
export PYTHONPATH=$PYTHONPATH:${PWD}/code
```